package.json

```
@@ -10,7 +10,7 @@
     "services:down": "docker compose -f infra/compose.yaml down",
     "lint:check": "prettier --check .",
     "lint:fix": "prettier --write .",
-    "test": "jest --runInBand",
+    "test": "npm run services:up && npm run wait-for-postgres && concurrently -n next,jest --hide next -k -s command-jest 'next dev' 'jest --runInBand'",
```

Seguindo o que o @aprendendofelipe disse sobre problemas ocasionados pelas aspas simples dependendo do sistema operacional. E complementando meu comentário no PR [filipedeschamps/clone-tabnews/pull/21](https://github.com/filipedeschamps/clone-tabnews/pull/21#issuecomment-2146095557)

```suggestion
    "concurrently": "concurrently -n next,jest --hide next -k -s command-jest \"next dev\" \"jest --runInBand\"",
    "test": "npm run services:up && npm run wait-for-postgres && npm run concurrently ",
```